surfactant/sbomtypes/_sbom.py

```diff
@@ -113,3 +114,272 @@ def create_software(
         )
         self.software.append(sw)
         return sw
+
+    def merge(self, sbom_m: "SBOM"):
+        merged_sbom = SBOM()
```

Comment:
Another way this could work is doing the merge into the `self` SBOM -- might simplify some parts of merging, since then we don't need to add entries from self into the new SBOM object.

Reply:
Thats true, I did it this way since I was translating the functionality as closely as I could to what was originally written. Would you like me to do this now, or create a followon ticket?

Reply:
Nvm it actually wasn't that bad to modify, made the changes! It now no longer starts the merge into an empty SBOM